src/app/components/LinkedData/index.tsx

```diff
@@ -111,12 +111,34 @@ const LinkedData = ({
 
   const hasByline = !!bylineLinkedData;
 
-  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
-    bylineLinkedData || {};
+  const places: string[] = [];
 
-  const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+  const getAuthorTagging = (data: BylineLinkedData | null) => {
+    const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
+      data || {};
 
-  const locationCreated = { '@place': location };
+    const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+    if (location) {
+      if (!places.includes(location)) {
+        places.push(location);
+      }
+    }
+
+    return {
+      '@type': 'Person',
+      name: authorName,
+      ...(sameAs.length && { sameAs }),
+      ...(authorImage && { image: authorImage }),
+    };
+  };
+  let bylineAuthors;
+  if (bylineLinkedData) {
+    bylineAuthors = bylineLinkedData?.map(data => getAuthorTagging(data));
+  }
```

Comment:
Can this be combined into a single line? (I've checked the syntax & `bylineAuthors` will be undefined if `bylineLinkedData` is undefined.  

```suggestion
  const bylineAuthors = bylineLinkedData?.map(data => getAuthorTagging(data));
```